File: BubbleSort.py
#Given n numbers sort them using Bubble Sort.
n=eval(input("Enter how many numbers"))
a=[]
for i in range(1,n+1):
  num=eval(input("Enter the number"))
  a.append(num)
  #Sorting
  for i in range(0,len(a)):
    for j in range(0,len(a)-1):
      if a[j]>a[j+1]:
        a[j],a[j+1]=a[j+1],a[j]
print(a)

File: BubbleSort.cpp
// Given n numbers, sort them using Bubble Sort
#include<iostream>
#include<fstream>
using namespace std;
int main()
{ int arr[10],n;
  cout<<"enter the number of elements in the array";
 cin>>n;
 cout<<"enter the elements"<<endl;
 for(int i=1;i<=n;i++)
 { 
   cin>>arr[i];
   cout<<endl;
 }
 for(int i=1;i<=n;i++)
 {
     for(int j=0;j<=n-1;j++)
     { if(arr[j]>arr[j+1])
     {
       int temp=arr[j];
       arr[j]=arr[j+1];
       arr[j+1]=temp;
     }
 }
}
 cout<<"the array in ascending order is";
 for(int i=0;i<n;i++)
 {
   cout<<arr[i];
 }
 
}
